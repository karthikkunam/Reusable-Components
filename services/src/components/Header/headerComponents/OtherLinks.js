import React from 'react';
import PropTypes from 'prop-types';

const OtherLinks = props => {
  const {
    link: { url, label },
  } = props;
  return (
    <ul className="navbar-nav">
      <li className="nav-item">
        <a className="nav-link header-name-text 7hub" href={url}>
          {label} <span className="sr-only">(current)</span>
        </a>
      </li>
    </ul>
  );
};



OtherLinks.propTypes = {
  link: PropTypes.objectOf({
    label: PropTypes.string,
    url: PropTypes.string,
  }),
};

export default OtherLinks;
