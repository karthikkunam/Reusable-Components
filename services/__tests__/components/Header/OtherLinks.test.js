import React from 'react';
import OtherLinks from '../../../src/components/Header/headerComponents/OtherLinks';
import { shallow } from 'enzyme';

describe('OtherLinks Component', () => {
  it('renders correctly', () => {
    const props = {
      link: {
        url: '',
        label: '',
      },
    };
    const component = shallow(<OtherLinks {...props} />);
    expect(component).toMatchSnapshot();
  });
});
