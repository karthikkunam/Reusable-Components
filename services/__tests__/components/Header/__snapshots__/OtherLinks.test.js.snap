// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OtherLinks Component renders correctly 1`] = `
<ul
  className="navbar-nav"
>
  <li
    className="nav-item"
  >
    <a
      className="nav-link header-name-text 7hub"
      href=""
    >
       
      <span
        className="sr-only"
      >
        (current)
      </span>
    </a>
  </li>
</ul>
`;
